src/pages/about.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: '../layouts/Legal.astro'
+title: 'Übersicht'
+---
+
+# Was Metropoliplan (MEP) ist
+
+## Zweck von Metropoliplan
+
+Metropoliplan wurde für die speziellen Bedürfnisse des öffentlichen Zeugnisgebens in Metropolregionen entwickelt. Zudem wurden auch Prozesse für das Zeugnisgeben auf Rastplätzen integriert.  
+
+Diese Software ist nicht für das Trolley Zeugnisgeben auf Versammlungsebene gedacht.
```

Comment:
~~Trolley Zeugnisgeben~~ Der Begriff lt. _sfl_ wäre "Zeugnisgeben in der Öffentlichkeit"